pom.xml

```diff
@@ -60,6 +60,7 @@
 
         <basepom.check.fail-all>true</basepom.check.fail-all>
 
+        <basepom.release.profiles>basepom.oss-release,oracle</basepom.release.profiles>
```

Comment:
@stevenschlansker This should prevent accidentally missing the `oracle` profile on release builds. You'll need to have followed the steps in `CONTRIBUTING.md` (at least the `settings.xml` and `settings-security.xml` stuff) before you can release again.